flow-typed/box-ui-elements.js

```diff
@@ -542,6 +542,32 @@ type Translations = {
     onTranslate?: Function
 };
 
+type OpenWithIntegrations = {
```

Comment:
Is this `items` wrapper necessary or can we track the array on state directly?

Reply:
In the future, the API may have a "default" integration, which will appear outside of this items wrapper. Going off our conversation earlier I should normalize everything into one array that we can keep in state.

Reply:
items are now a part of the API response but not what we store in state on the client.